scripts/hudson/narayana.sh

```diff
@@ -416,13 +416,9 @@ function build_as {
 
 function tests_as {
   # running WildFly testsuite if configured to be run by axis AS_TESTS
-  if [ $AS_TESTS_TRACE ]; then
```

Comment:
the change removes the ability to set trace output when running the AS_TESTS axis as the server is not built and ran in two steps now. But the test passes when it's ran in one step though rather than seeming to cause every build to fail